src/components/constants/Items.ts

```diff
@@ -408,6 +408,226 @@ export class Items {
         "Town Map": {
             "effect": "Shows your position in the Pokemon World",
             "category": "Key"
-        }
+        },
+        "HM01": {
+            "effect": "Cut",
```

Comment:
What if there's a move named `["A", "b", "c", "FemaleSymbol"]`? How do you plan on dealing with that?

I think you'll need to change the item schema a bit.